package com.ecommerce.exception;
public class UserUtility {

	public static final String USER_NOT_EXIST = "User is not THERE";
	public static final int USER_NOT_EXIST_STATUS = 609;
	

	
	public static final String BOUGHT_SUCCESSSFULLY = "User Bought Successfully!";
	public static final int BOUGHT_SUCCESSSFUL_STATUS = 612;
	
	public static final String BOUGHT_UNSUCCESSSFULL = " unSuccessfull!";
	public static final int BOUGHT_UNSUCCESSSFUL_STATUS = 613;
	
	public static final String PRODUCT_NOT_EXIST = "Product is not THERE";
	public static final int PRODUCT_NOT_EXIST_STATUS = 606;
	
	
	public static final String NO_PRODUCT_PRESENT = "No Product for this User";
    public static final int NO_PRODUCT_PRESENT_ERRORCODE = 777;
    
   
    
    public static final String SUCCESSSFULLY_SAVED_PRODUCT_DATA = "Product saved Successfully!";
    public static final int SUCCESSSFULLY_SAVED_PRODUCT_DATA_STATUS = 701;
    
	public static final String ORDER_NOT_EXIST = "Order does not Exist";
	public static final int ORDER_NOT_EXIST_STATUS = 705;

	private UserUtility() {

	}

}
